### PR TITLE
BLD: eggtart with single example protos; training scripts

### PR DESCRIPTION
**Pull request by eigenfoo**

### CODE REVIEW

**Comment by eigenfoo** [comment]
To experiment:

- [ ] Only first conv layer should be coordconv.
- [ ] Rethink pooling and batch norms... do they really make sense?

File: src/train.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    plt.imshow(confusion)
+    plt.colorbar()
+    classes = class_index.keys()
+    tick_marks = np.arange(len(classes))
+    plt.xticks(tick_marks, classes, rotation=45)
+    plt.yticks(tick_marks, classes)
+    plt.ylabel('True label')
+    plt.xlabel('Predicted label')
+    plt.tight_layout()
+    plt.savefig('confusion_{}.png'.format(epoch_num))
```
Comment (ccurro) [inline src/train.py]: Save to unique dir?

Comment (eigenfoo) [reply src/train.py]: What should we name it? I've just been in the habit of appending the latest committish to the names of all output files.
